features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/utils/url-validation/url-validation.js

```diff
@@ -1,7 +1,7 @@
 $(document).ready(function(){
     $( "body" ).delegate( "button.check_url_valid", "click", function() {
         var btn = this;
-        var url = $(this).parent().find('input:first').val();
+        var url = $(this).parent().parent().find('input:first').val();
```

Comment:
Don't traverse elements like this. Use `parent()` api if you want to traverse single element. Else use `closest()`.